internal/e2e/main_test.go

```diff
@@ -808,6 +858,28 @@ func TestMinioGroupBinding(t *testing.T) {
 
 		WaitForStableResourceVersion(td, gb)
 	})
+
+	t.Run("can still delete a minio group binding when user deleted", func(t *testing.T) {
```

Comment:
e2e test that reproduces a similar issue in `MinioGroupBinding` resources and verifies that the issue is fixed